### PR TITLE
Add LocationService Info to the Readme EDGECLOUD-3723

### PR DESCRIPTION
**Pull request by ah1053**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -259,8 +259,24 @@ MobiledgeX Unity Package comes with  WebSocket Implementation (MobiledgeXWebSock
...
  ```
 
+## Location
+MobiledgeX SDK uses a combination of device Location  & MCC-MNC code to connect you to the closet Edge data center where your backend is deployed.
 
+The SDK comes with an easy to integrate Location Service Solution (LocationService.cs), LocationService.cs must be added to the Scene in order for the SDK to automatically ask for Location Permission and use the user location.
 
+You can find LocationService in the inspector,Select AddComponent then select (MobiledgeX/LocationService)
+
+![](https://developers.mobiledgex.com/assets/unity-sdk/mobiledgex-unity-location-service.png)
```
Comment (vmohanx) [inline README.md:269]: This seems to be a missing file - is there a different PR that adds this?

File: README.md [hunk 2]
```diff
@@ -259,8 +259,24 @@ MobiledgeX Unity Package comes with  WebSocket Implementation (MobiledgeXWebSock
  
  ```
 
+## Location
+MobiledgeX SDK uses a combination of device Location  & MCC-MNC code to connect you to the closet Edge data center where your backend is deployed.
 
+The SDK comes with an easy to integrate Location Service Solution (LocationService.cs), LocationService.cs must be added to the Scene in order for the SDK to automatically ask for Location Permission and use the user location.
 
+You can find LocationService in the inspector,Select AddComponent then select (MobiledgeX/LocationService)
```
Comment (vmohanx) [inline README.md:267]: switch, for .

File: README.md [hunk 3]
```diff
@@ -259,8 +259,24 @@ MobiledgeX Unity Package comes with  WebSocket Implementation (MobiledgeXWebSock
...
+
+![](https://developers.mobiledgex.com/assets/unity-sdk/mobiledgex-unity-location-service.png)
+
+If you have different Location flow or for testing use the following:
```
Comment (vmohanx) [inline README.md:271]: different way to get the device's location

File: README.md [hunk 4]
```diff
@@ -259,8 +259,24 @@ MobiledgeX Unity Package comes with  WebSocket Implementation (MobiledgeXWebSock
  
  ```
 
+## Location
+MobiledgeX SDK uses a combination of device Location  & MCC-MNC code to connect you to the closet Edge data center where your backend is deployed.
```
Comment (vmohanx) [inline README.md:263]: instead of device Location, "GPS Location"

File: README.md [hunk 5]
```diff
@@ -259,8 +259,24 @@ MobiledgeX Unity Package comes with  WebSocket Implementation (MobiledgeXWebSock
...
 
+The SDK comes with an easy to integrate Location Service Solution (LocationService.cs), LocationService.cs must be added to the Scene in order for the SDK to automatically ask for Location Permission and use the user location.
```
Comment (vmohanx) [inline README.md:265]: Maybe add here that Location Service uses GPS

**Comment by ah1053** [comment]
Changes are already approved on dev center

**Pull request merged**